Update dock popup tab color for disabled slots

aaronp64 opened the pull request:
Updated tab color for disabled slots in dock popup to match the dock area color

Fixes #113313

<!--
Please target the `master` branch in priority.

Relevant fixes are cherry-picked for stable branches as needed by maintainers.

To speed up the contribution process and avoid CI errors, please set up pre-commit hooks locally:
https://contributing.godotengine.org/en/latest/engine/guidelines/code_style.html
-->

AdriaandeJongh reviewed (approved):
LGTM. didn’t look at the code.

aaronp64 committed:
Update dock popup tab color for disabled slots

Updated tab color for disabled slots in dock popup to match the dock area color
